spock-tests/src/test/groovy/com/rackspacecloud/blueflood/repose/ReposeSpec.groovy

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2024 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.repose
+
+import groovy.json.JsonBuilder
+import org.apache.http.client.methods.HttpGet
+import org.apache.http.client.methods.HttpPost
+import org.apache.http.entity.StringEntity
+import org.apache.http.impl.client.HttpClients
+import spock.lang.IgnoreIf
+import spock.lang.Shared
+import spock.lang.Specification
+
+/*
+* Repose upgrade activity needs to validate various Repose filters features
+* This test can be run against local vagrant or stage or prod blueflood endpoints
+*
+* Vagrant (local) allows us to configure repose filters to tweak as needed.
+* Example: Rate limits, authorization needs
+*
+* Pre-requisites for running:
+* Environment variables:
+*  - TENANT_ID
+*  - X-Auth-Token - Rackspace identity token to work with keystone-v2 filter config
+*  - RATE_LIMIT_COUNT (Optional) - to test against vagrant or staging with configurable rate limit filters
```

Comment:
This doesn't explain what this argument does. What happens when you set a "rate limit count"?

Reply:
Updated